aten/src/ATen/mps/MPSGuardImpl.mm

```diff
@@ -9,9 +9,6 @@
   void MPSGuardImpl::createEvent(
     mpsEvent_t* event,
     const EventFlag flag) const {
-    id<MTLDevice> mtl_device = MPSDevice::getInstance()->device();
```

Comment:
Wait what is this function doing now? Both args are ignored and it is a no-op??

Reply:
I don't know what its doing, but even before the change it was a no-op, wasn't it?
But I guess it was supposed to do something like `*event = new mpsEvent(flag);`?

Reply:
It was indeed a noop before so this is ok. But we should fix this for sure!